TableSchemerExamples/MasterViewController.swift

```diff
@@ -109,6 +109,10 @@ class MasterViewController: UITableViewController {
                         print("Selected object in ArrayScheme: \(object)")
                         self.tableView.deselectRow(at: self.tableView.indexPathForSelectedRow!, animated: true)
                     }
+
+                    scheme.reorderingHandler = { objects in
+                        print("Reordered objects in ArrayScheme: \(objects)")
```

Comment:
Can you also update the arrayObjects value? I think it'll be good to show updating the backing objects.